test/unit/services/sshAdapter_spec.js

```diff
@@ -0,0 +1,357 @@
+"use strict";
+
+var AWS        = require("aws-sdk");
+var sinon      = require("sinon");
+var expect     = require("chai").expect;
+var SshAdapter = require("../../../lib/services/sshAdapter");
+var Bluebird   = require("bluebird");
+var options    = {};
+var ec2        = new AWS.EC2();
```

Comment:
Is there any reason that the `options` and the `ec2` instance shouldn't be in the `describe` block? I don't think they will ever get garbage collected if they stay out here.